tests/manifest-path-encoding.unit.test.ts

```
@@ -0,0 +1,57 @@
+import { describe, expect, it } from 'vitest'
+
+import { normalizeLegacyEncodedManifestFileName } from '../supabase/functions/_backend/utils/manifest_encoding.ts'
+import { getManifestUrl } from '../supabase/functions/_backend/utils/downloadUrl.ts'
```

ESLint

[error] Expected "../supabase/functions/_backend/utils/downloadUrl.ts" to come before "../supabase/functions/_backend/utils/manifest_encoding.ts".

(perfectionist/sort-imports)